doc/api/repl.markdown

```diff
@@ -237,3 +237,27 @@ The following key combinations in the REPL have these special effects:
   - `<ctrl>D` - Similar to the `.exit` keyword.
   - `<tab>` - Show both global and local(scope) variables
 
+
+### Customizing Object displays in the REPL
+
+The REPL module internally uses
+[util.inspect()][], when printing values. But, `util.inspect` delegates the call
+ to the object's `inspect` function, if it has one. You can read more about
+ this delegation [here][].
+
+If you have defined an `inspect` function on an object, like this:
```

Comment:
Should this be referred to as `inspect()`? I wish we had guidelines for that kind of thing... cc @nodejs/documentation